rust/main/agents/scraper/src/db/block.rs

```diff
@@ -97,7 +97,7 @@ impl ScraperDb {
         trace!(?models, "Writing blocks to database");
         match Insert::many(models)
             .on_conflict(
-                OnConflict::column(block::Column::Hash)
+                OnConflict::columns([block::Column::Domain, block::Column::Height])
```

Comment:
I wonder if it will identify a conflict over (Domain, Hash), but different heights?